### PR TITLE
#fix working example/basic/api function

### PR DESCRIPTION
**Pull request by AntonDzyk**
API functions in example/basic not working.
I'm fix it.
i think it will be useful in the example.

### CODE REVIEW

**Comment by AntonDzyk** [comment]
@easonlin404 hello! Eason Lin, please, can you see my PR?

File: example/basic/docs/docs.go [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+// This file was generated by swaggo/swag
 
 package docs
 
 import (
+	"bytes"
+	"encoding/json"
+	"strings"
+
+	"github.com/alecthomas/template"
```
Comment (easonlin404) [inline example/basic/docs/docs.go:11]: could I know why need to introduce `github.com/alecthomas/template` pkg?

Comment (AntonDzyk) [reply example/basic/docs/docs.go:11]: I created docs/docs.go by calling the command `swag init` (version 1.6.7)
I can minimize changes to the file if needed.

Comment (easonlin404) [reply example/basic/docs/docs.go:11]: I got it. It generated by command swag init. I didn't notice. 😆

**Comment by easonlin404** [comment]
@AntonDzyk Thanks for your effort on it. 💯

**Pull request merged**